SPECS/python-markupsafe/python-markupsafe.spec

```diff
@@ -20,7 +20,7 @@ BuildRequires:  python3-setuptools
 BuildRequires:  python3-xml
 Requires:       python3
 Provides:       python3dist(markupsafe) = %{version}-%{release}
-Provides:       python3.7dist(markupsafe) = %{version}-%{release}
+Provides:       python3.9dist(markupsafe) = %{version}-%{release}
```

Comment:
Suggestion: do you think we could use a macro here, which would always extract the version number from `%{python3_version}` and place that number here, so we don't miss any future updates?